src/views/Home.css

```
@@ -28,12 +28,38 @@
 	margin: 0;
 }
 
-.intro-text {
+.Home-app-name {
+	color: #5a23b3;
+	font-size: 45px;
+}
```

```suggestion
      text-shadow:
      -1px -1px 0 white,
      1px -1px 0 white,
      -1px 1px 0 white,
      1px 1px 0 white;
}
```
Adding this creates a thin white stroke around the app name so it's legible in dark mode.